plugin-testing-core/src/main/java/com/palantir/gradle/plugintesting/GradleTestVersions.java

```diff
@@ -30,7 +30,7 @@
  */
 public final class GradleTestVersions {
     static final String TEST_GRADLE_VERSIONS_SYSTEM_PROPERTY = "TEST_GRADLE_VERSIONS";
-    static final List<String> DEFAULT_TEST_GRADLE_VERSIONS = Arrays.asList("7.6.4", "8.8");
+    static final List<String> DEFAULT_TEST_GRADLE_VERSIONS = Arrays.asList("8.14.3");
```

Comment:
Really, we never should have had default versions like this - I'm not sure how these got here. Regardless, let's update it for now until we get around to fixing this ecosystem.